ml-agents/mlagents/trainers/policy/torch_policy.py

```diff
@@ -98,59 +98,56 @@ def export_memory_size(self) -> int:
 
     def _split_decision_step(
         self, decision_requests: DecisionSteps
-    ) -> Tuple[SplitObservations, np.ndarray]:
-        vec_vis_obs = SplitObservations.from_observations(decision_requests.obs)
+    ) -> Tuple[List[np.array], np.ndarray]:
+        obs = decision_requests.obs
```

Comment:
This feels weird now. Whatever calls _split_decision_step should just get the `obs` from `DecisionSteps` instead.

Reply:
and _split_decision_step should just return the mask

Reply:
I renamed the method and it now only returns the masks